.github/workflows/publish_pypi_wheel.yml

```diff
@@ -0,0 +1,78 @@
+name: PyPI Publish Wheels
+
+on:
+  workflow_dispatch:
```

Comment:
Maybe we can TEST release when we push things to main. Usually it happens before release, wdyt?

Reply:
ah, i see what you mean - we push to testPyPi on push to main. Yeah, that's actualyl quite cool! Let me introduce it!